### PR TITLE
BL-1731: Add generic booster.

### PR DESCRIPTION
**Pull request by dkinzer**
Adds new macro for boosting based on labels.

Adds label for inverse boosting on bad_libraries label.
Adds label for inverse boosting on bookseller label.

### CODE REVIEW

**Comment by ebtoner** [comment]
@dkinzer Some labeling suggestions:
- Can you rename "bad_libraries" to "no_boost_libraries"? 
- Are there other ways to make clear that what we're actually doing is **lowering** the boost for the labels?

**Comment by dkinzer** [comment]
@ebtoner we're still boosting it though.  It's just we are using an inverse boost.

**Comment by ebtoner** [comment]
Ok, can you clarify that it's an inverse boost in that case?

**Comment by dkinzer** [comment]
@ebtoner it's any 1/x boost where x is greater than 1.  0.5 would half the score for the record if boost label is matched.

**Comment by ebtoner** [comment]
@dkinzer I understand what is functionally happening. I'm just asking for slightly clearer labels / documentation for this. At a minimum, please update "bad_libraries" to something more neutral.

**Comment by dkinzer** [comment]
OK, I think I'll change the labels to inverse_boost_*. Does that work?

**Comment by ebtoner** [comment]
Perfect! thanks!

File: lib/cob_index/macros/booster.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    lambda do |rec, acc|
+      add_libraries_labels(rec, acc)
+      add_booksellers_labels(rec, acc)
+    end
+  end
+
+  private
+
+    def add_libraries_labels(rec, acc)
+      bad_libraries = [ "PRESSER", "CLAEDTECH" ]
```
Comment (ebtoner) [inline lib/cob_index/macros/booster.rb:21]: Please edit to "low_priority_libraries" here and elsewhere in this PR

Comment (dkinzer) [reply lib/cob_index/macros/booster.rb]: @ebtoner I decided to just remove that line.

**Review by ebtoner — APPROVED** [review]

**Pull request merged**